client-rpc/src/bookmap/bookmap.py

```diff
@@ -625,13 +638,13 @@ def resize_order(addon: typing.Dict[str, object],
         _stop_addon()
 
 
-def subscribe_to_indicator(addon: typing.Dict[str, object], addon_name: str, alias: str,
```

Comment:
I have a concern here. This method actually can subscribe to the add-on provider as well as to the provider's generator. Maybe it will be better to just create another method with a name like subscribe_to_provider and just put the same logic underneath, so it will be clearer for users what the method is doing.